lib/models/src/main/kotlin/no/nav/amt/lib/models/deltaker/DeltakerEndring.kt

```diff
@@ -0,0 +1,92 @@
+package no.nav.amt.lib.models.deltaker
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import no.nav.amt.lib.models.arrangor.melding.Forslag
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class DeltakerEndring(
+    val id: UUID,
+    val deltakerId: UUID,
+    val endring: Endring,
+    val endretAv: UUID,
+    val endretAvEnhet: UUID,
+    val endret: LocalDateTime,
+    val forslag: Forslag?,
+) {
+    data class Aarsak(
+        val type: Type,
+        val beskrivelse: String? = null,
+    ) {
+        init {
+            if (beskrivelse != null && type != Type.ANNET) {
+                error("Aarsak $type skal ikke ha beskrivelse")
+            }
+        }
+
+        enum class Type {
+            SYK,
+            FATT_JOBB,
+            TRENGER_ANNEN_STOTTE,
+            UTDANNING,
+            IKKE_MOTT,
+            ANNET,
+        }
+    }
+
+    @JsonTypeInfo(use = JsonTypeInfo.Id.SIMPLE_NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
+    sealed class Endring {
```

Comment:
NIT: Om jeg kan kommentere litt personlig preferanse så foretrekker jeg egentlig å ikke nøste for mange dataklasser sånn at strukturen da blir flatere og filene ikke blir så store å scrolle i. Da har man har egne filer for ting med navngivning som er helt konkret for hva det gjelder. For eksempel her så syns jeg det hadde vært ryddigere å ha en egen fil for dataklassen Endring. 
Da blir man litt tvunget til å konkretisere hva er "Endring". Vi har jo allerede "DeltakerEndring".

Reply:
I dette tilfellet synes jeg det er ryddigere å ha det nøstet på denne måten siden vi også har `Endring` som en del av `Forslag` (de er nesten like, men ikke helt). Da er man avhengig av konteksten for å forstå hvilken endring det er snakk om - den som er en del av forslag eller den som er en del av deltakerendring.

Reply:
Trenger ikke fikse alt mulig nå, det kan jo være forskjellige meninger om og dette er vel copy paste

Reply:
De to klassene (denne og den for forslagene) er ikke like da, så det blir ikke bare copy-paste 😅 Når vi har to endring-klasser synes jeg det er enklere å holde oversikt og bruke riktig klasse når den ene er Forslag.Endring og den andre er DeltakerEndring.Endring. Det blir jo dessuten vanskelig å finne gode navn for dem hvis de skal leve frittstående uten kobling til der de brukes. Men ja, det går an å endre på ting senere også så lenge man holder endringene bakoverkompatible.